test/e2e/l7networkpolicy_test.go

```diff
@@ -289,6 +293,71 @@ func testL7NetworkPolicyHTTP(t *testing.T, data *TestData) {
 	})
 }
 
+func testL7NetworkPolicyHTTPLargeResponse(t *testing.T, data *TestData) {
+	skipIfNumNodesLessThan(t, 2)
+	clientPodName := "test-l7-http-large-resp-client-selected"
+	clientPodLabels := map[string]string{"test-l7-http-large-resp-e2e": "client"}
+
+	// Create a client Pod, with the Pod being selected by the test L7 NetworkPolicy as target.
+	require.NoError(t, NewPodBuilder(clientPodName, data.testNamespace, agnhostImage).OnNode(nodeName(0)).WithLabels(clientPodLabels).Create(data))
+	_, err := data.podWaitForIPs(defaultTimeout, clientPodName, data.testNamespace)
+	require.NoError(t, err, "Expected IP for Pod '%s'", clientPodName)
+
+	// Create a hostNetwork server Pod as the destination selected by the test L7 NetworkPolicy, ensuring test traffic
+	// traverses antrea-gw0.
+	serverPodName := "test-l7-http-large-resp-server"
+	cmd := []string{"/agnhost", "netexec", "--http-port=8081"}
+	require.NoError(t, NewPodBuilder(serverPodName, data.testNamespace, agnhostImage).
+		WithHostNetwork(true).
+		OnNode(nodeName(1)).
+		WithCommand(cmd).
+		Create(data))
+	podIPs, err := data.podWaitForIPs(defaultTimeout, serverPodName, data.testNamespace)
+	require.NoError(t, err, "Expected IP for Pod '%s'", serverPodName)
+	serverIPs := podIPs.AsSlice()
+
+	l7ProtocolAllowsPathShell := []crdv1beta1.L7Protocol{
+		{
+			HTTP: &crdv1beta1.HTTPProtocol{
+				Path: "/shell*",
+			},
+		},
+	}
+	// Create a test egress L7NetworkPolicy allowing HTTP path "shell*".
+	policyAllowPathShellName := "test-l7-http-allow-path-shell"
+	createL7NetworkPolicy(t, data, false, policyAllowPathShellName, 1, nil, clientPodLabels, ProtocolTCP, p8081, l7ProtocolAllowsPathShell)
+	time.Sleep(networkPolicyDelay)
+
+	// Get the MTU of the test client Pod, assuming it's the MTU of the K8s cluster.
+	mtuStdout, _, err := data.RunCommandFromPod(data.testNamespace, clientPodName, agnhostContainerName, []string{"cat", "/sys/class/net/eth0/mtu"})
+	require.NoError(t, err)
+	mtu, err := strconv.Atoi(strings.TrimSpace(mtuStdout))
+	require.NoError(t, err)
+
+	for _, ip := range serverIPs {
+		baseURL := net.JoinHostPort(ip.String(), strconv.Itoa(int(p8081)))
```

Comment:
ditto, you can use `"8081"` instead of `strconv.Itoa(int(p8081))`

Reply:
Done